src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -112,6 +126,19 @@ const TreeItem: React.FC<TreeItemProps> = props => {
     return style;
   }, [isMatchItem]);
 
+  React.useEffect(() => {
```

Comment:
This hook will not be needed if we save the remote to the redux state, see my other comment.